src/Admin/BaseBlockAdmin.php

```diff
@@ -38,15 +38,9 @@ abstract class BaseBlockAdmin extends AbstractAdmin
      */
     protected $blockManager;
 
-    /**
-     * @var bool
-     */
-    protected $inValidate = false;
+    protected bool $inValidate = false;
 
-    /**
-     * @var array
-     */
-    protected $containerBlockTypes = [];
+    protected array $containerBlockTypes = [];
```

Comment:
This change and the one above, should be reverted too. (not the one below).

Reply:
Could you explain me why this change have an impact. Is there something I didn't understand?

Reply:
oh. sorry. I will fix all these out-of-scope changes